Move node management related guides from Core docs - Closes #561 (#629)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/index.adoc

= Guides overview
Mona Bärenfänger <mona@lightcurve.io>
:description: This section provides an overview of the main guides in chronological order, together with an example using the 'Hello World' App.
:toc:
:imagesdir: ../../assets/images
:url_github_hello: https://github.com/LiskHQ/lisk-sdk-examples/tree/development/hello_world
:url_react: https://reactjs.org/

:url_config: guides/app-development/configuration.adoc
:url_custom: guides/app-development/customize.adoc
:url_frontend: guides/app-development/frontend.adoc
:url_launch: guides/app-development/launch.adoc
:url_api_access: guides/node-management/api-access.adoc
:url_forging: guides/node-management/forging.adoc
:url_enable_ssl: guides/node-management/enable-ssl.adoc
:url_logging: guides/node-management/logging.adoc
:url_setup: setup.adoc
:url_tutorials: tutorials/index.adoc

This section consists of concise, goal-oriented how-to guides that are designed in a manner to help the user solve specific issues.

== Application development

> The guides are listed in chronological order, walking the user through the basic steps of developing a blockchain application with the Lisk SDK.

Reviewer: ```suggestion
The guides in the application development section are aimed towards developers, who want to built their own blockchain application with the Lisk SDK.

The guides are listed in chronological order, walking the user through the basic steps of developing a blockchain application by the example of the <<hello_world_app, Hello World app>>.

=== Guides overview (application development)
```

. The xref:{url_config}[configuration guide] covers how to configure a blockchain application and how to modify the default values.
. The xref:{url_custom}[custom transactions guide] explains how to add and register a new transaction.
. The xref:{url_broadcast}[broadcast a transaction guide] describes how to create a transaction object of a specific transaction type, and how to broadcast it to the network.
. The xref:{url_frontend}[frontend guide] covers how to create, setup, and run a frontend or client app that interacts with a Lisk blockchain network.
. The xref:{url_launch}[launch guide] describes how to publish a blockchain application, which will enable other users to join the network, and will turn your local 1-node Devnet into an actual blockchain network.
. The xref:{url_interact}[interact guide] covers how to interact with a blockchain network powered by the Lisk SDK, communicate with the API of a node, including examples of common interactions, for the transactions, blocks and accounts.
. The xref:{test}[test guide] explains how to write unit tests required for transaction verification, including a working example.

> 

Author: OK, agreed and now done :-)

image::run_a_blockchain_10_secs.gif[Logs of a node]

All remaining topics of the Hello World application are explained further in the dedicated Lisk SDK guides.

== Node management

> The node management guides are designed to assist the user by covering all the relevant aspects, right from providing recommendations to monitoring and debugging a node.

Reviewer: ```suggestion
The node management guides are aimed towards admins and delegates, who operate a node that runs with the Lisk SDK.
Each of these guides covers a specific topic of node management, gives examples, and explains the available options.
```

The most important sections pertaining to the node management are described below:

> . The xref:{url_api_access}[API access guide] covers the options as well as how to control the access to a node, including the configurable flags.

Reviewer: ```suggestion
. The xref:{url_api_access}[API access guide] explains, how to manage the API access to a node.
```
